modules/mauth-proxy/README.md

```diff
@@ -1,35 +1,45 @@
-## Goal:
-
-The goal of this simple service is to append valid mAuth headers to any incoming request and send this (now authenticated) request to the destination service.
-
-## General use:
-Using [Typesafe Config](https://github.com/typesafehub/config)
-Create `application.conf` file on your classpath with following settings
-```json
-proxy {
-  port: 9090
-  port: ${?PROXY_PORT}
-  buffer_size_in_bytes: 524288
-  buffer_size_in_bytes: ${?BUFFER_SIZE_IN_BYTES}
-}
-
-app {
-  uuid: ${?APP_UUID}
-  private_key: ${?APP_PRIVATE_KEY}
-}
+# Mauth Proxy:
+The reason for using the mAuth proxy is to enable the use of UI REST clients like 
+[Postman](https://www.getpostman.com/postman), [Paw](https://paw.cloud/) or curl to request Medidata resources.
+
+The service appends valid mAuth headers to incoming requests and 
+sends the authenticated request to the requested service.
+
+## Running via sbt:
+```bash
+export APP_MAUTH_UUID="<the mauth id of the app you are proxying to>"
+export APP_MAUTH_PRIVATE_KEY="<the actual key (not the path) of the app you are proxying to>"
+./runMauthProxyServer.bash
 ```
 
-## Start the proxy with Docker
+## Running via Docker image
+
+Internal Medidata users please follow [internal guide](https://mdsol.jiveon.com/community/research-development/app-dev/platform-development/services/mauth/blog/2018/03/06/use-mauthed-service-with-a-mauth-unaware-client)
+
+### Installing Docker
+You need to have Docker installed on your machine in order to use Mauth Proxy. The simplest way to get Docker running 
+on a Mac or Windows machine is to follow the instructions at 
+[Docker for Mac](https://docs.docker.com/#/docker-for-mac).
```

Comment:
The text says Mac or Windows, but this is a link specifically to the Mac instructions. Maybe change this to a generic Docker installation link?